filament/include/filament/View.h

```diff
@@ -150,6 +152,7 @@ class UTILS_PUBLIC View : public FilamentAPI {
         float anamorphism = 1.0f;                //!< Bloom x/y aspect-ratio (1/32 to 32)
         uint8_t levels = 6;                      //!< number of blur levels (3 to 12)
         BlendMode blendMode = BlendMode::ADD;    //!< How the bloom effect is applied
+        bool threshold = true;                   //!< Whether to threshold the source
```

Comment:
Do we really want to enable this by default? Grmbl grmbl

Reply:
We can definitely revisit if it causes issues, so far it looks better imho (it doesn't soften the image, while giving more control on the bloom strength). At low 'strength' it's basically moot.